imports/ui/components/parties/party_page.html

```diff
@@ -1,32 +1,34 @@
 <template name='partyPage'>
-  <div class='party-link'>
-    <a href='/party'>← all parties</a>
-    {{#if ownParty}}
-      <a href='/events/edit/{{slug}}'><button type="button" class="btn btn-default btn-md party-header__button">
-      <span class="glyphicon glyphicon-edit" aria-hidden="true"></span> Edit</button></a>
-    {{/if}}
-  </div>
   {{#if Template.subscriptionsReady}}
     {{#with party}}
-      <div class='flyer-div'>
-        <img class='flyer-div__front' src='{{flyerFront.url}}'>
-        {{#if flyerBack.url}}
-          <img class='flyer-div__back' src='{{flyerBack.url}}'>
-        {{/if}}
+      <h4 class='news__title'>{{title}}</h4>
+      <div class='show__link'>
+        <a href='/events' class='show__link-back'>← all events</a>
+        <!-- {{#if ownParty}}
+          <a href='/events/edit/{{slug}}'><button type="button" class="btn btn-default btn-md party-header__button">
+          <span class="glyphicon glyphicon-edit" aria-hidden="true"></span> Edit</button></a>
+        {{/if}} -->
       </div>
-      <div class='party-info'>
-        <h3><b>{{title}}</b></h3>
-        <h4>{{location}}</h4>
-        <h5>{{time startTime}}</h5>
-        <p>{{description}}</p>
-        <p class='party__tag'>{{#each tag in tags}}#{{tag}}&nbsp;{{/each}}</p>
-        <div class='party-info__details'>
-          <span class='party-info__star glyphicon glyphicon-star {{upvoted upvoters}}'></span><span class='party-info__upvotes-count'>{{upvoteCount}}</span>
+      <div class='news-item'>
```

Comment:
Again it's pretty awkweird having the news styles all over the party template. Maybe we could add selectors like "party__title" to their corresponding CSS objects, for clarity. What do you think?

Reply:
I'd say we should merge the selectors into a universal one and use it in place of the selectors.

Reply:
That sounds good. Would you pick a suitable name and edit the news file to match, please? Thanks very much, Derek. I am sorry I have been a poor correspondent, by the way. Let's talk on Thursday some time. I'll be free all day and maybe in the evening, too. Aloha.